pages/result.html

```diff
@@ -19,6 +21,24 @@
         console.log(Kakao.isInitialized());
     </script>
 
+    <!-- For Twitter -->
+    <script>window.twttr = (function (d, s, id) {
+            var js, fjs = d.getElementsByTagName(s)[0],
+                t = window.twttr || {};
+            if (d.getElementById(id)) return t;
+            js = d.createElement(s);
+            js.id = id;
+            js.src = "https://platform.twitter.com/widgets.js";
+            fjs.parentNode.insertBefore(js, fjs);
+
+            t._e = [];
+            t.ready = function (f) {
+                t._e.push(f);
+            };
+
+            return t;
+        }(document, "script", "twitter-wjs"));</script>
+
```

Comment:
Can't we put this in javscript file?

Reply:
Currently this code is unnecessary because we are not using multiple Twitter widgets. In fact, we are using none since we use Twitter web intent to share via tweet. So we can delete it if you want, but this is good enough place for this line of code if we want it there. Here's the link to the document: https://developer.twitter.com/en/docs/twitter-for-websites/javascript-api/guides/set-up-twitter-for-websites

Reply:
Ok. Let's clean up the code at last step.